contrib/devtools/proto-tools-installer.sh

```diff
@@ -85,9 +85,9 @@ f_install_buf() {
 
 f_install_protoc_gen_gocosmos() {
   f_print_installing_with_padding protoc-gen-gocosmos
-  pushd "${TEMPDIR}" >/dev/null
+  # Note: we need to run this command in the root folder, because we added some
+  # an alias to our own fork of gogoproto inside go.mod.
```

Comment:
can we check current dir and report warning/error here?

Reply:
If you add this to `go.mod` please ensure `go.sum` does not change.

Reply:
> can we check current dir and report warning/error here?

done 👍 

@alessio I'm not sure what you mean, I'm not changing go.mod in this PR.

Reply:
I've miscommunicated my thoughts, sorry about that. If you don't `cd`/`pushd` into another directory before running `go get`, your command will likely affect `go.{mod,sum}`. So I meant: please just ensure that the script remains deterministic/idempotent and does not affect go module dependencies - if that makes sense.

Reply:
Understood, that makes sense.

I ran this command, and it does not change `go.{mod,sum}` today. But how can I be sure it won't change those files in the future?

I thought about adding a `git checkout HEAD -- go.mod`, but it's not ideal, as the user may have installed other go modules prior to running this command, and this git will undo those changes.

Reply:
In theory `go get github.com/regen-network/cosmos-proto@v0.3.0` should just do. We just need to ensure that `v0.3.0` matches the version in `go.mod`.